docs/sql-programming-guide.md

```diff
@@ -1087,6 +1087,13 @@ the following case-sensitive options:
   </tr>
 
   <tr>
+     <td><code>maxConnections</code></td>
+     <td>
+       The number of JDBC connections, which specifies the maximum number of simultaneous JDBC connections that are allowed. This option applies only to writing. It defaults to the number of partitions of RDD.
```

Comment:
At the end of the description, I can add it. But, this will look different with other options in terms of level of depth. Is it okay?